_config.yml

```diff
@@ -27,4 +29,3 @@ description: A community-driven fork of Magento Community Edition
 google_analytics: UA-62512293-1
 show_downloads: true
 theme: jekyll-theme-cayman
-
```

Comment:
having an empty line at the end of files is a best practice
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file